lib/Refactoring/LocalRename.cpp

```diff
@@ -204,10 +205,13 @@ class RenameRangeCollector : public IndexDataConsumer {
 
   RenameRangeCollector(const ValueDecl *D)
       : stringStorage(new StringScratchSpace()) {
-    SmallString<64> SS;
-    llvm::raw_svector_ostream OS(SS);
-    printValueDeclUSR(D, OS);
-    usr = stringStorage->copyString(SS.str());
+    SmallString<64> usrStorage;
+    llvm::raw_svector_ostream usrOS(usrStorage);
+    printValueDeclUSR(D, usrOS);
+    usr = stringStorage->copyString(usrStorage.str());
+
+    llvm::raw_svector_ostream nameOS(displayName);
+    index::printDisplayName(D, nameOS);
```

Comment:
Do we really have to expose `printDisplayName` here? It just handles getters/setters, which presumably we don't care about here anyway (ie. can this just be `D->getName()`?)

Reply:
I would prefer to yes. Less because we need it right now and more to be future-proof. If we modify `printDisplayName` in the future without remembering to change the call here, we would be missing results from related idents.